service/src/chain_spec.rs

```diff
@@ -171,9 +171,7 @@ fn staging_testnet_config_genesis() -> polkadot::GenesisConfig {
 				.collect(),
 		}),
 		indices: Some(polkadot::IndicesConfig {
-			ids: endowed_accounts.iter().cloned()
-				.chain(initial_authorities.iter().map(|x| x.0.clone()))
-				.collect::<Vec<_>>(),
+			indices: vec![],
```

Comment:
Why remove this?

Reply:
Indicies are now an "opt in" thing.

So by default, accounts won't have them.